src/state/zustand/swapperStore/amountSelectors.ts

```diff
@@ -0,0 +1,394 @@
+import { createSelector } from 'reselect'
+import { TradeAmountInputField } from 'components/Trade/types'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { fromBaseUnit, toBaseUnit } from 'lib/math'
+import type { SwapperState } from 'state/zustand/swapperStore/useSwapperStore'
+
+const selectAssetPriceRatio = createSelector(
+  (state: SwapperState) => state.buyAssetFiatRate,
```

Comment:
all of these can be pulled out/aliased if you prefer for visual cleanliness but no big deal either way

Reply:
Added convenience selectors in https://github.com/shapeshift/web/pull/4138/commits/f30f3dffcecdcd3834a422c5cfff66ef084220a5